mcstatus/protocol/connection.py

```diff
@@ -285,31 +289,32 @@ def __del__(self):
 
 
 class UDPAsyncSocketConnection(AsyncReadConnection):
-    stream = None
-    timeout = None
+    # These will only be None until connect is called, ignore the None type assignment
+    stream: asyncio_dgram.aio.DatagramClient = None  # type: ignore[assignment]
+    timeout: float = None  # type: ignore[assignment]
```

Comment:
```suggestion
    stream: Optional[asyncio_dgram.aio.DatagramClient] = None
    timeout: Optional[float] = None
```

Reply:
I assume this can also be marked as resolved following the discussion on why I didn't use optional above.